hal/generic/TimerServiceGeneric.cpp

```diff
@@ -49,11 +50,10 @@ namespace hal
                 nextTrigger = infra::TimePoint::max();
 
                 infra::EventDispatcher::Instance().Schedule([this]()
-                {
+                    {
                     std::unique_lock<std::recursive_mutex> eventLock(mutex);
```

Comment:
This needs an additional level of indentation

Reply:
Setting the ColumnLimit fixes this; but with same consequence as above.